redbot/core/commands/converters.py

```diff
@@ -0,0 +1,74 @@
+import re
+from datetime import timedelta
+from . import Converter, Context, BadArgument
```

Comment:
I know this package is a bit confusing due to the way it imports all of discord.py's stuff to the top level, then writes our stuff afterwards.

When importing something we've overwritten from within the actual `commands` package, import it from the sibling-module, not from the top level. So in this example, it would be `Context`, you want to do `from .context import Context`.

Also, whenever you're doing a relative import which is only for a type-hint, please use [`typing.TYPE_CHECKING`](https://docs.python.org/3.7/library/typing.html#typing.TYPE_CHECKING) and forward references to avoid any circular imports.

Reply:
okay for `Context`, but what about `Converter` and `BadArgument`? Should I import `discord.ext.commands` instead?